Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/BaseIpcLib.c

```diff
@@ -0,0 +1,131 @@
+/** @file
+  Base IPC library implementation.

```

Comment:
Need to use standard BSD 2 + Patent copyright header.

Reply:
Same comment for all new files.

Reply:
would you be able to review the commit here: andreyv1978#1